src/main/java/az/gdg/mssubscriber/model/dto/SubscriberDTO.java

```diff
@@ -0,0 +1,13 @@
+package az.gdg.mssubscriber.model.dto;
+
+import lombok.*;
```

Comment:
Please review my comments for this pr: https://github.com/GDG-Baku/ms-auth/pull/1
I am repeating my comment : 
The use of * is considered a bad practice. You can read the explanation here, for example: https://stackoverflow.com/questions/14897448/use-of-in-import-statement-in-java